src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Snowflake integration
+tags:
+    - Snowflake integration
+    - New Relic integrations
+metaDescription: Install our Snowflake dashboards and see your Snowflake data in New Relic.
+---
+
+import infrastructureSnowflakeDashboard from 'images/infrastructure_screenshot-full_snowflake-dashboard.png'
+
+With our Snowflake dashboards, you can easily track query performance and logins, monitor your Snowflake warehouses, and get a total overview of the health of your storage system, like potential security incidents. Built with our infrastructure agent and our Snowflake integration, Snowflake dashboards let you view your most critical query data, all in one place.
+
+<img
+    src={infrastructureSnowflakeDashboard}
+    title="Snowflake dashboard"
+    alt="A screenshot of a dashboard with Snowflake query metrics."
+/>
+<figcaption>
+After setting up the Snowflake integration with New Relic, your data will display in dashboards like these, right out of the box. 
+</figcaption> 
+
+## Install the infrastructure agent [#infra]
+
+Before getting Snowflake data into New Relic, first install our infrastructure agent. This allows New Relic to collect data from Snowflake so performance metrics, logins, and other data appear in dashboards. 
+
+There are two methods to install the infrastructure agent:
+
+* Follow our [guided install](https://one.newrelic.com/nr1-core?account=3501165&state=ff641d8b-e722-855e-4c4a-c50c23969dc3) to instrument your system with the infrastructure agent. 
```

Comment:
we often have very little info about the difference between guided install and regular infra install. not saying you need to work on this now, as i see this in many places. something I sent to paperclypse about this: 

i keep seeing various new docs say 'use our guided install' and link directly to UI. I wonder if we might improve our guided install doc https://docs.newrelic.com/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview/ and go into more detail about why to use it and what rules/limitations are and instead link to that for more context. I can see many customers being confused around why they'd use guided install and if we send them straight to UI that doesn't really help.